src/mc_tasks/lipm_stabilizer/StabilizerTask.cpp

```diff
@@ -981,9 +982,9 @@ void StabilizerTask::distributeWrench(const sva::ForceVecd & desiredWrench)
   // |w_l_0 + w_r_0 - desiredWrench|^2
```

Comment:
@mmurooka I think we should change this comment to reflect the modified equation and either explain the reasoning (numeric issue as we move away from the origin?) or link to this PR?

Reply:
Thanks, I will merge this after the CI lands!